integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -106,7 +105,7 @@ def _transmit_create(self, channel_metadata_item_map, content_updated_mapping):
         for chunk in islice(chunk_items, transmission_limit):
             serialized_chunk = self._serialize_items(list(chunk.values()))
             try:
-                self.client.create_content_metadata(serialized_chunk)
+                response_status_code, response_body = self.client.create_content_metadata(serialized_chunk)
```

Comment:
most clients and remote systems treat the success/failure of a whole chunk. in theory, we may later have systems which support a more granular status within that chunk. we may also need that more granular status to track remote content. in that case, we may offer an optional second `create_content_metadata` type method which accepts the chunk of items rather than the `serialized_chunk` and individually track the status, remote content id created, etc.